tests/validation/tests/v3_api/test_aks_cluster.py

```diff
@@ -60,9 +60,9 @@ def get_aks_version(multiple_versions=False):
         json_response = json.loads(response.content)
 
         if multiple_versions and len(json_response) > 1:
-            aksclusterversion = [json_response[0], json_response[-1]]
+            aksclusterversion = [json_response[0], json_response[-2]]
```

Comment:
would you mind adding a comment as to why the order is what it is / what these are?

Reply:
Sure, I'll go ahead and do that.